pages/advanced/content/fr/index.js

```diff
@@ -10,4 +10,6 @@ module.exports = {
     require('./size'),
     require('./ssr'),
   ].filter(mod => Boolean(mod.title)),
+  contributorsTitle: 'Contributeurs:',
```

Comment:
Actually, I _had_ an idea, and it wasn't good and was complicated. Then I thought of my other idea which I think you can totally do (even though you say you're not too familiar with React :wink:). You're doing a great job! Don't stop now! :) I think you know where to take things from here :)